core/src/main/java/org/apache/spark/SparkFirehoseListener.java

```diff
@@ -34,12 +34,12 @@ public void onEvent(SparkListenerEvent event) { }
 
     @Override
     public final void onStageCompleted(SparkListenerStageCompleted stageCompleted) {
-        onEvent(stageCompleted);
+      onEvent(stageCompleted);
```

Comment:
Yeah, the whole file is mis-indented. The first indent is still 4 spaces. I think you could fix that too if you like